src/base.jl

```diff
@@ -0,0 +1,51 @@
+
+################################################################################
+# Julia => Python
+################################################################################
+api_map = Dict(:decision_function => :decision_function,
+               :fit_predict! => :fit_predict,
+               :fit_transform! => :fit_transform,
+               :get_feature_names => :get_feature_names,
+               :get_params => :get_params,
+               :predict => :predict,
+               :predict_proba => :predict_proba,
+               :predict_log_proba => :predict_log_proba,
+               :partial_fit! => :partial_fit,
+               :score_samples => :score_samples,
+               :sample => :sample,
+               :score => :score,
+               :transform => :transform,
+               :inverse_transform => :inverse_transform,
+               :set_params! => :set_params)
+               
+for (jl_fun, py_fun) in api_map
+    @eval $jl_fun(py_model::Py, args...; kwargs...) =
+        tweak_rval(py_model.$(py_fun)(args...; kwargs...))
+end
+
+fit!(py_model::Py, args...; kwargs...) = py_model.fit(args...; kwargs...)
+
+################################################################################
+# Functions
+################################################################################
+clone(py_estimator::Py; safe::Bool=true) = sklearn.base.clone(py_estimator, safe=safe)
+is_classifier(py_estimator::Py) = pyconvert(Bool, sklearn.base.is_classifier(py_estimator))
```

Comment:
Is there a difference between a "py_estimator" and a "py_model"?

Reply:
There is not a difference, it was just inconsistent naming. I have updated them all to be `py_model`